src/main/java/frc/robot/UltrasonicSensor.java

```diff
@@ -0,0 +1,23 @@
+package frc.robot;
+
+import edu.wpi.first.wpilibj.AnalogInput;
+
+public class UltrasonicSensor {
+  AnalogInput input;
+
+  public UltrasonicSensor(AnalogInput input) {
+    this.input = input;
+  }
+
+  /**
+   * Get current pressure in PSI.
+   * @return double pressure in PSI.
+   */
```

Comment:
This comment wasn't updated when this class was copied! 😱

Reply:
Fixed.